### ISSUE TITLE
Material Symbols: Rotate 90 Degrees CCW has No Filled Icon

### ISSUE THREAD

**Issue by JaxonWright** [google/material-design-icons#1450]
As [seen](https://fonts.google.com/icons?icon.style=Filled&icon.set=Material+Icons&icon.query=rotate+90) in the documentation for material, this icon does not have a filled variant. The one for rotating 90 degrees clockwise does, however.

https://fonts.google.com/icons?icon.style=Filled&icon.set=Material+Icons&icon.query=rotate+90

**Comment by tphinney** [comment]
This is true for the old Material Icons set. It was fixed in the newer Material Symbols, though.

**Comment by fil146** [comment]
Hey @tphinney!
I would like to add that shapes are also different, please take a look on the screenshot:
![image](https://github.com/user-attachments/assets/2a7a674f-7edc-4b3c-87b3-2a6c5e754085)

**Comment by tphinney** [comment]
This is bug, but a completely separate issue. Can you create a new issue for it?

(Or I can—but if you do it, then you will get notifications etc.)

**Comment by fil146** [comment]
@tphinney hey!
Thanks for your advise, I've created the issue: https://github.com/google/material-design-icons/issues/1749
